src/functions/videos.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require("node-fetch");
+
+const apiKey = process.env.GOOGLE_API_KEY;
+
+exports.handler = async (videoId) => {
```

Comment:
@chihab 
I don't know how Netlify functions work, but I did create a videos function following the the same pattern so it can works using server tool (`tools/server.js`).
I'm sure that the way I did to get `videoId` parameter will not work on Netlify server, as I said, it works for server.js tool.
Can you take care of this ?

Reply:
Sure!
Thanks @ikourfaln

Reply:
Please accept this PR https://github.com/ikourfaln/ngx-darija/pull/1 on your branch

Reply:
Done